bahmni-emr-api/src/main/java/org/openmrs/module/bahmniemrapi/encountertransaction/mapper/ETObsToBahmniObsMapper.java

```diff
@@ -84,6 +84,10 @@ protected BahmniObservation map(EncounterTransaction.Observation observation, Ad
         return bahmniObservation;
     }
 
+    private boolean validateFormNameSpace(EncounterTransaction.Observation observation) {
+        return observation.getFormNamespace() == null && CONCEPT_DETAILS_CONCEPT_CLASS.equals(observation.getConcept().getConceptClass());
+    }
+
```

Comment:
could we have test to support this change? if there were no test could you add one?

Reply:
sure @N0-man